src/components/views/settings/FontScalingPanel.tsx

```diff
@@ -75,6 +75,15 @@ export default class FontScalingPanel extends React.Component<IProps, IState> {
         });
     }
 
+    public async componentDidUpdate(): Promise<void> {
```

Comment:
This should be replaced with:
- Set the state and start watching settings by calling `SettingsStore.watchSetting` in `componentDidMount`
- `watchSetting` returns a ref that can be used to unwatch in `componentWillUnmount`